### ISSUE TITLE
Mouse cursor appearance configuration is present twice

### ISSUE THREAD

**Issue by stefonarch** [lxqt/lxqt#658]
"Mouse theme selection" is actually  possible to do in two different places: 
once in  "appearance" which is ok and another time in "mouse and keyboard", where only functional settings should belong.

**Comment by jleclanche** [comment]
This is OK and completely acceptable.

**Comment by pmattern** [comment]
Distributing different kinds of features to the various GUIs `lxqt-config-*`, here limiting `lxqt-config-input` to technical aspects, makes perfect sense, providing the cursor theme's configuration at two different locations is confusing and doesn't offer a single advantage.
Subjective evaluation of mine, but I'm [not alone](https://github.com/lxde/lxqt/issues/793#issuecomment-135236929).

Solving this issue involves just removal of stuff and hence doesn't mean too much effort.

Decisions based on subjective evaluation shouldn't be taken by a single person whoever it is, should they?

So all in all I'd really suggest to have another look at this one.

**Comment by jleclanche** [comment]
> Decisions based on subjective evaluation shouldn't be taken by a single person, whoever it is, should they?

Please don't understand this as me taking a final decision. Decisions are only as final as the devs want them to be, so if you feel strongly about this, you're free to reopen it.

With that said, I maintain my stance. Both places are somewhere users would expect them to be. Fact is, when you hit Alt+F2 and type "Mouse", what comes up is *only* `lxqt-config-input`.

I'd be willing to compromise on a `Configure cursor theme` button to launch the cursor theme config at the bottom of the "Mouse" section of `lxqt-config-input`.

**Comment by micrococo** [comment]
I was about to open a new issue for this. I think that cursor theme selection is out of place in lxqt-config-input. Almost no desktop includes this option in the input device dialog, and even if they do (Xfce for example) they don't repeat the same configuration option in the appearance dialog. So I think it should be removed in either the appearance dialog or preferably in the input dialog.

**Comment by tsujan** [comment]
As far as coding is concerned, I agree that having two instances of cursor settings might cause troubles at some point. But I think it's worth probable troubles because, as far as *the user* is concerned, the cursor settings should be accessible in both places.

**Comment by tsujan** [comment]
As a compromise, a way of avoiding two instances *of the code* (and so, avoiding probable future troubles) can be as follows:

 1. Add a command-line option to `lxqt-config-appearance` for showing the cursor page.
 2. Add a button to `lxqt-config-input` → Cursor, so that if it's clicked, the cursor page of `lxqt-config-appearance` is shown.

**Comment by tsujan** [comment]
Oh, sorry, I got so confused, although I was working on lxqt-config's code! There's no extra code. Both pages use the same code. So, IMHO, there's nothing to fix here.

**Comment by tsujan** [comment]
It's only one code, and the cursor is applied by one and the same method: `applyCusorTheme()`.

The difference is only that, in `lxqt-config-appearance`, `ConfigOtherToolKits` will also be called if GTK is enabled. I don't think we should remove it from input setting just because of GTK.

I repeat my opinion, which may not be liked by many: *LXQt shouldn't interfere with GTK; GTK settings should be set by using GTK tools like `dconf-editor`.*

**Comment by stefonarch** [comment]
We need to keep average users in mind, we have already WM/Compositor settings to considerate separately. I bet there are users who just have no idea about Qt and Gtk and even less about dconf-editor - but they will see that their cursor changes.

**Comment by tsujan** [comment]
Perhaps we could call `ConfigOtherToolKits` here if needed. But removing this section just because of GTK would be an excessive change. If we want to pretend that GTK isn't an alien to LXQt, at least we shouldn't harm Qt.

**Comment by isf63** [comment]
I agree with @stefonarch that it should be easy to configure GTK, despite LXQt being a Qt "environment".  Two prime examples of giant GTK apps are Firefox and Thunderbird.

**Comment by tsujan** [comment]
I don't disagree with applying the setting to GTK — as I said, it may be possible in this case (and I have a patch) — what I'm saying is that there's no reason to remove the cursor theme section from `lxqt-config-input` only because of that.

**Comment by isf63** [comment]
Because it's implemented as a shared library there is no code duplication, which would be the main problem.

**Comment by tsujan** [comment]
All done :) I'm making a PR and adding your name to reviewers.
